dub.json

```diff
@@ -94,6 +94,28 @@
                 "source/agora/utils/gc/*"
             ]
         },
+        {
+            "name": "ut",
+            "preBuildCommands": ["$DUB run --compiler=$$DC unit-threaded -c gen_ut_main -- -f bin/ut.d -d $DUB"],
+            "mainSourceFile": "bin/ut.d",
+            "excludedSourceFiles": [
+                "source/agora/cli/*/main.d",
+                "source/agora/utils/gc/*"
+            ],
```

Comment:
That should be `unittest`, why a separate config ?

Reply:
It was to be able to run unit tests with or without using unit-threaded. But I had problems with getting it to work anyway.